src/ci-manager/templates/config.yaml

```diff
@@ -13,17 +13,18 @@ data:
   MANAGER_AUTHORITY: harness-manager:9879
   GRPC_SERVER_PORT: {{.Values.service.grpcport | quote}}
   SCM_SERVICE_URI: "scm-service:8091"
-  ADDON_IMAGE: {{ include "common.images.image" (dict "imageRoot" .Values.ci_images.addon.image "global" .Values.global) }}
-  LE_IMAGE: {{ include "common.images.image" (dict "imageRoot" .Values.ci_images.lite_engine.image "global" .Values.global) }}
-  GIT_CLONE_IMAGE: {{ include "common.images.image" (dict "imageRoot" .Values.ci_images.git_clone.image "global" .Values.global) }}
-  DOCKER_PUSH_IMAGE: {{ include "common.images.image" (dict "imageRoot" .Values.ci_images.kaniko.image "global" .Values.global) }}
-  ECR_PUSH_IMAGE:  {{ include "common.images.image" (dict "imageRoot" .Values.ci_images.kaniko_ecr.image "global" .Values.global) }}
-  GCR_PUSH_IMAGE: {{ include "common.images.image" (dict "imageRoot" .Values.ci_images.kaniko_gcr.image "global" .Values.global) }}
-  GCS_UPLOAD_IMAGE: {{ include "common.images.image" (dict "imageRoot" .Values.ci_images.gcs_upload.image "global" .Values.global) }}
-  S3_UPLOAD_IMAGE: {{ include "common.images.image" (dict "imageRoot" .Values.ci_images.s3_upload.image "global" .Values.global) }}
-  ARTIFACTORY_UPLOAD_IMAGE: {{ include "common.images.image" (dict "imageRoot" .Values.ci_images.artifactory_upload.image "global" .Values.global) }}
-  GCS_CACHE_IMAGE: {{ include "common.images.image" (dict "imageRoot" .Values.ci_images.gcs_cache.image "global" .Values.global) }}
-  S3_CACHE_IMAGE: {{ include "common.images.image" (dict "imageRoot" .Values.ci_images.s3_cache.image "global" .Values.global) }}
+  ADDON_IMAGE: '{{ .Values.ci_images.addon.image.repository }}:{{ .Values.ci_images.addon.image.tag }}'
```

Comment:
where will the registry information flow from ? through UI/Connector ?

Reply:
On the NG UI, under account level, we create a docker connector. That connector adds a prefix to all the images being used in CI.
As for details,how it is done. The CI team suggested this change to be done.